adcircpy/forcing/winds/best_track.py

```diff
@@ -25,12 +25,31 @@
 from pyproj import CRS, Proj
 from shapely import ops
 from shapely.geometry import Point, Polygon
+from math import sin, cos, sqrt, atan2, radians
```

Comment:
Using numpy is preferred, because numpy can operate over arrays, math can only operate over scalars. Essentially, if you use numpy, you can compute all of the values in a single go (vectorized form) instead of doing a for loop, which is much slower.